mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb

```diff
@@ -169,6 +169,12 @@ resource "google_project_service" "acm" {
   provider = google-beta
 }
 
+resource "google_project_service" "poco" {
+  project = google_project.project.project_id
+  service = "anthospolicycontroller.googleapis.com"
+  provider = google-beta
```

Comment:
You should enable it for GA provider if you are running your tests against the GA provider